Frontend/cypress/integration/rentals.spec.js

```diff
@@ -1,32 +1,17 @@
 /// <reference types="cypress" />
 import data from "../../spec/Database/DummyData/rentals";
 import columns from "../../src/components/TableEditors/Rentals/Columns";
-import { dateToString, waitForPopupToClose, clearFilter } from "./utils";
+import { dateToString, waitForPopupToClose, clearFilter, isAtSameDay } from "./utils";
+import COLORS from "../../src/components/Input/ColorDefs";
 
 let rentals;
 let currentRentals;
 
-function millisAtStartOfDay(millis) {
-  var msPerDay = 86400 * 1000;
-  return millis - (millis % msPerDay);
-}
+const TODAY = Date.UTC(2020, 0, 1);
 
-function isBeforeDay(m1, m2) {
-  return millisAtStartOfDay(m1) < millisAtStartOfDay(m2);
-}
-
-function isToday(millis) {
-  return millisAtStartOfDay(millis) === Date.UTC(2020, 0, 1);
-}
-
-function isBeforeToday(millis) {
-  return isBeforeDay(millis, Date.UTC(2020, 0, 1));
```

Comment:
how does this work?

Reply:
These functions were used to detect whether a time (given in ms) is at a certain day. Therefore millisAtStartOfDay calculates the time in ms at the start of a day (00:00). Then these start times can be compared to each other in order to detect if both are at the same day or not.